cgroup2/manager.go

```diff
@@ -529,7 +528,10 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 	if err != nil {
 		return nil, err
 	}
-	out := make(map[string]interface{})
+	// Sizing this avoids an allocation to increase the map at runtime;
+	// currently the default bucket size is 8 and we put 40+ elements
+	// in it so we'd always end up allocating.
+	out := make(map[string]string, 50)
```

Comment:
perhaps have the size as 0 and capacity as 64?

Reply:
You can't set the capacity of a map or chan, only slices

Reply:
Right, my bad, ENEEDCOFFEE